lib/Chat/ChatManager.php

```diff
@@ -730,26 +725,14 @@ public function getMessagesById(Room $chat, array $commentIds): array {
 		return $comments;
 	}
 
-	public function queueRemindLaterBackgroundJob(Room $chat, IComment $comment, Attendee $attendee, int $timestamp): void {
-		$this->jobList->add(ChatMessageReminder::class, [
-			'execute-after' => $timestamp,
-			'token' => $chat->getToken(),
-			'message_id' => $comment->getId(),
-			'message_actor_type' => $comment->getActorType(),
-			'message_actor_id' => $comment->getActorId(),
-			'user' => $attendee->getActorId(),
-		]);
-	}
-
-	public function dismissReminderNotification(Room $chat, IComment $comment, Attendee $attendee): void {
-		$notification = $this->notificationManager->createNotification();
-		$notification->setApp(Application::APP_ID)
-			->setUser($attendee->getActorId())
-			->setObject('reminder', $chat->getToken())
-			->setMessage('reminder', [
-				'commentId' => $comment->getId(),
-			]);
-		$this->notificationManager->markProcessed($notification);
+	/**
+	 * Get messages by ID
+	 *
+	 * @param int[] $commentIds
+	 * @return IComment[] Key is the message id
+	 */
+	public function getMessagesById(array $commentIds): array {
```

Comment:
Could also be like this to enforce type: 
```suggestion
	public function getMessagesById(int ...$commentIds): array {
```
If accepted, maybe in a future pull request to uniformize other methods like `getCommentsById` or `getMessagesForRoomId`